doc/source/MaintainersSection.rst

```diff
@@ -241,22 +241,21 @@ Create modulefile ``/work2/06146/tg854455/frontera/spack-stack/modulefiles/git-l
 UW (Univ. of Wisconsin) S4
 ------------------------------
 
+gnu (module only)
+   The ``gnu/9.3.0`` module provided by the system administrators is broken (circular dependencies etc.). To create a usable version, copy ``/data/prod/hpc-stack/modulefiles/core/gnu/9.3.0.lua`` into directory ``/data/prod/jedi/spack-stack/modulefiles/gnu`.`
+
 miniconda
    Follow the instructions in :numref:`Section %s <Prerequisites_Miniconda>` to create a basic ``miniconda`` installation and associated modulefile for working with spack. Don't forget to log off and back on to forget about the conda environment.
 
-qt (qt@5)
-   The default ``qt@5`` in ``/usr`` is incomplete and thus insufficient for building ``ecflow``. After loading/unloading the modules as shown below, refer to 
-   :numref:`Section %s <Prerequisites_Qt5>` to install ``qt@5.15.2`` in ``/data/prod/jedi/spack-stack/qt-5.15.2``.
```

Comment:
I think this is what I'm getting at with my comment in S4 packages.yaml file. Should the instructions in Prerequisites.rst for building `qt@5.15.2` be removed?

Reply:
No, because there are still some other systems that may need them. The prerequisites section should probably have a disclaimer at the top that not all of this is needed, depending on the system. It's the maintainers section that tells people what to build on which system.

Reply:
Got it. Thanks for the explanation!. I'm good with leaving qt instructions in the prerequisites section. I think your disclaimer idea is good, but I'm considering that optional and will approve now. Thanks!